fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStats.java

```diff
@@ -180,23 +262,10 @@ public void updateStats(Type columnType, Map<StatsType, String> statsTypeToValue
         }
     }
 
-    public List<String> getShowInfo() {
-        List<String> result = Lists.newArrayList();
-        result.add(Long.toString(ndv));
-        result.add(Float.toString(avgSize));
-        result.add(Long.toString(maxSize));
-        result.add(Long.toString(numNulls));
-        if (minValue != null) {
-            result.add(minValue.getStringValue());
-        } else {
-            result.add("N/A");
-        }
-        if (maxValue != null) {
-            result.add(maxValue.getStringValue());
-        } else {
-            result.add("N/A");
-        }
-        return result;
+    // TODO: The generated statistics should not be modified
+    public void updateBySelectivity(double selectivity) {
+        ndv = (long) Math.ceil(ndv * selectivity);
```

Comment:
Why not use `Math.round`?

Reply:
It was previously implemented to update ndv by selectivity. the way I understand it is that `Math.round` might lead result to 0.

I intend to remove this method, the collected statistics should not be changed in this way.

Reply:
ok thx